### PR TITLE
update style 

### PR DESCRIPTION
**Pull request by faizalanwar**

### CODE REVIEW

**Comment by faizalanwar** [comment]
Kindly Add `hacktoberfest-accepted` label

File: css/style.css [hunk 1]
```diff
@@ -98,21 +98,23 @@ button {
...
 }
 
 .btn1 {
   width: 19vw;
-  max-width: 250px;
-  font-weight: 600;
-  border-radius: 30px;
+  max-width: 380px;
+  font-weight: 700;
+  border-radius: 9999px;
```
Comment (Zack-Dx) [inline css/style.css]: Sorry< what's this?

File: css/style.css [hunk 2]
```diff
@@ -98,21 +98,23 @@ button {
 }
 button:hover {
   background-color: rgb(222, 222, 222);
-  border: 3px solid rgb(222, 222, 222);
+  border-color: rgb(51, 54, 57);
+  border-width: 1px;
```
Comment (Zack-Dx) [inline css/style.css]: U can write this is one line

**Review by Zack-Dx — CHANGES_REQUESTED** [review]

**Comment by faizalanwar** [comment]
ok i will try to fix it

**Pull request closed without merging**